SygicMapsKit/Classes/SYMKSdkManager.swift

```diff
@@ -0,0 +1,35 @@
+import SygicMaps
+
+public typealias SdkInitializationCompletion = (_ success: Bool)->()
+
+public class SYMKSdkManager {
```

Comment:
Pridal by som `private init()` kedze je to singleton, alebo ked init nieje uvedeny sprava sa ako private?

Reply:
private init, public shared